plugin.rb

```diff
@@ -228,7 +228,10 @@ def self.github_organizations
 
       scope format: true, constraints: { format: 'json' } do
         resources :organizations, only: [:index] do
-          resources :repos, only: [:index] do
+
+          # need to allow dots in the id, use the same username
+          # regex from core
+          resources :repos, only: [:index], id: /[%\w.\-]+?/ do
```

Comment:
Don't we need to escape `.` here too? Otherwise, it is treated as any single character in a regexp.

Reply:
Not sure...I figured if it works for usernames in core it should be fine? Works in regexr as well:

![image](https://user-images.githubusercontent.com/920448/122870540-2296b780-d371-11eb-9597-573823fbd006.png)

Reply:
Hmm maybe in a `[]` it is smart enough to know that we mean dot. Haha probably still work it to just escape it to avoid confusion.

Reply:
I tried to find rules around what characters are available for a repo, couldn't find an official guide. I decided to try it - Weird characters are replaced with `_` `-` or `.`.

<img width="324" alt="Screen Shot 2021-06-22 at 3 51 47 pm" src="https://user-images.githubusercontent.com/72780/122871055-d304bb80-d371-11eb-918f-b421b13e4485.png">

Reply:
I just got rid of %